### ISSUE TITLE
Can't install on CentOS 9 through repo

### ISSUE THREAD

**Issue by yowkah** [Xpra-org/xpra#3600]
**Describe the bug**
When attempting to install xpra through repo as instructed on the website there is a missing dependency, it looks like xpra-html5 is not present in the repo.

**To Reproduce**
1. fresh install of CentOS 9 Stream
2. import gpg keys with `sudo rpm --import https://xpra.org/gpg.asc`
3. add the repo using `sudo wget -O /etc/yum.repos.d/xpra.repo https://xpra.org/repos/CentOS/xpra.repo`
4. run `sudo dnf install xpra`

error is as shown:
```
Error: 
 Problem: cannot install the best candidate for the job
  - nothing provides xpra-html5 needed by xpra-4.4-10.r31612.el9.x86_64
(try to add '--skip-broken' to skip uninstallable packages or '--nobest' to use not only best candidate packages)
```

**System Information (please complete the following information):**
 - Server OS: CentOS Stream 9
 - Client OS:  not relevant
 - Xpra Server Version: 4.4-10.r31612.el9.x86_64
 - Xpra Client Version: not relevant

**Additional context**
Manually browsing the repo i don't seem to find xpra-html5 listed in there. Also trying to build an rpm from xpra-html5 as instructed on that repo results in errors with the rpmspec:

```
removing 'xpra-html5-6.0' (and everything under it)
Traceback (most recent call last):
  File "/home/jouke/Downloads/xpra-html5/./setup.py", line 594, in <module>
    sys.exit(main(sys.argv))
  File "/home/jouke/Downloads/xpra-html5/./setup.py", line 583, in main
    make_rpm()
  File "/home/jouke/Downloads/xpra-html5/./setup.py", line 496, in make_rpm
    proc = Popen(["rpmspec", "-q", "--rpms", SPEC], stdout=PIPE, stderr=PIPE)
  File "/usr/lib64/python3.9/subprocess.py", line 951, in __init__
    self._execute_child(args, executable, preexec_fn, close_fds,
  File "/usr/lib64/python3.9/subprocess.py", line 1821, in _execute_child
    raise child_exception_type(errno_num, err_msg, err_filename)
FileNotFoundError: [Errno 2] No such file or directory: 'rpmspec'
```

**Comment by totaam** [comment]
My bad, I haven't pushed `xpra-html5` builds to the `9` repos yet.
For the time being, I've just copied the RPM package from `8` there - it **should** work until I get a chance to schedule a rebuild.

> `FileNotFoundError: [Errno 2] No such file or directory: 'rpmspec'`

You need to install `rpmspec` - I will add this to the instructions or make it optional.
IIRC, on RHEL 9 this may need to be done using `pip install rpmspec`.

**Comment by yowkah** [comment]
Thanks for the fast reply. Just checked the install, it got further but hangs on a different point now:

```
warning: Signature not supported. Hash algorithm SHA1 not available.
Last metadata expiration check: 0:02:07 ago on Thu 04 Aug 2022 10:14:03 AM CEST.
Error: 
 Problem: package xpra-4.4-10.r31282.el9.x86_64 requires python3-xpra-client = 4.4-10.r31282.el9, but none of the providers can be installed
  - package python3-xpra-client-4.4-10.r31282.el9.x86_64 requires python3-xpra = 4.4-10.r31282.el9, but none of the providers can be installed
  - cannot install the best candidate for the job
  - nothing provides libturbojpeg.so.0()(64bit) needed by python3-xpra-4.4-10.r31282.el9.x86_64
  - nothing provides libturbojpeg.so.0(TURBOJPEG_1.0)(64bit) needed by python3-xpra-4.4-10.r31282.el9.x86_64
  - nothing provides libturbojpeg.so.0(TURBOJPEG_1.2)(64bit) needed by python3-xpra-4.4-10.r31282.el9.x86_64
  - nothing provides libturbojpeg.so.0(TURBOJPEG_1.4)(64bit) needed by python3-xpra-4.4-10.r31282.el9.x86_64
  - nothing provides python3-lz4 needed by python3-xpra-4.4-10.r31282.el9.x86_64
  - nothing provides turbojpeg needed by python3-xpra-4.4-10.r31282.el9.x86_64
(try to add '--skip-broken' to skip uninstallable packages or '--nobest' to use not only best candidate packages)
```

tried with --no-best but unfortunately it didn't work yet.

**Comment by totaam** [comment]
> enabling the CRB repo from CentOS 9 which contains libturbo

Yes, starting with 9, they've moved some libraries to other repositories.
Looks like they have also moved `python3-lz4`..

**Comment by totaam** [comment]
We have a specfile for `python3-lz4`: https://github.com/Xpra-org/xpra/blob/master/packaging/rpm/python3-lz4.spec

This may work for you.
(I seem remember that there may be issues with it like missing build libraries causing problems)

The commit above adds it to the packages to build for `9` releases so the next round of builds should produce packages... or errors. We'll see.

**Comment by totaam** [comment]
All the packages should now be in the stable and beta repositories.
The problem with `python-lz4` was only for `aarch64` and has been resolved: #6

Please let me know if I have missed something else.

**Comment by totaam** [comment]
For the record, the lz4 issues are now irrelevant since we no longer use `python-lz4`: #3601
